Sources/SmileID/Classes/Networking/Models/Services.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+
+// MARK: - ServicesResponse
+
+public struct ServicesResponse: Codable {
+    public let bankCodes: [BankCode]
+    public let hostedWeb: HostedWeb
+
+    enum CodingKeys: String, CodingKey {
+        case bankCodes = "bank_codes"
+        case hostedWeb = "hosted_web"
+    }
+}
+
+// MARK: - BankCode
+
+public struct BankCode: Codable {
+    let name, code: String
+}
+
+// MARK: - HostedWeb
+
+public struct HostedWeb: Codable {
+    var basicKyc: CountryCodeToCountryInfo
+    var biometricKyc: CountryCodeToCountryInfo
+    var enhancedKyc: CountryCodeToCountryInfo
+    var docVerification: CountryCodeToCountryInfo
+    var enhancedKycSmartSelfie: CountryCodeToCountryInfo
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        basicKyc = try container.decode(CountryCodeToCountryInfo.self, forKey: .basicKyc)
+        biometricKyc = try container.decode(CountryCodeToCountryInfo.self, forKey: .biometricKyc)
+        enhancedKyc = try container.decode(CountryCodeToCountryInfo.self, forKey: .enhancedKyc)
+        docVerification = try container.decode(CountryCodeToCountryInfo.self, forKey: .docVerification)
+        enhancedKycSmartSelfie = try container.decode(CountryCodeToCountryInfo.self, forKey: .enhancedKycSmartSelfie)
+
+        basicKyc = basicKyc.toCountryInfo()
+        biometricKyc = biometricKyc.toCountryInfo()
+        enhancedKyc = enhancedKyc.toCountryInfo()
+        docVerification = docVerification.toCountryInfo()
+        enhancedKycSmartSelfie = enhancedKycSmartSelfie.toCountryInfo()
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case basicKyc = "basic_kyc"
+        case biometricKyc = "biometric_kyc"
+        case enhancedKyc = "enhanced_kyc"
+        case docVerification = "doc_verification"
+        case enhancedKycSmartSelfie = "ekyc_smartselfie"
+    }
+}
+
+// MARK: - CountryInfo
+/**
+ * The countryCode  field is not populated/returned by the API response, hence it being marked as
+ * mutable. However, it should be populated before usage of this class.when the response gets decoded
+ *the same applies to availableIdTypes
```

Comment:
nit: spacing/punctuation

```suggestion
 * The countryCode field is not populated/returned by the API response, hence it being marked as
 * mutable. However, it should be populated before usage of this class/when the response gets decoded.
 * The same applies to availableIdTypes
```